include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -510,7 +510,8 @@ struct SessionOptionsImpl : Base<T> {
   SessionOptionsImpl& SetCustomThreadCreationOptions(void* ort_custom_thread_creation_options);      ///< Wraps OrtApi::SessionOptionsSetCustomThreadCreationOptions
   SessionOptionsImpl& SetCustomJoinThreadFn(OrtCustomJoinThreadFn ort_custom_join_thread_fn);        ///< Wraps OrtApi::SessionOptionsSetCustomJoinThreadFn
 
-  void RegisterCustomOpsLibrary(const ORTCHAR_T* library_name);  ///< Wraps OrtApi::RegisterCustomOpsLibrary_V2
+  void RegisterCustomOpsLibrary(const ORTCHAR_T* library_name);         ///< Wraps OrtApi::RegisterCustomOpsLibrary_V2
+  void RegisterCustomOpsUsingFunction(const ORTCHAR_T* function_name);  ///< Wraps OrtApi::RegisterCustomOpsUsingFunction
```

Comment:
should it be `const char* function_name`?

Reply:
should it return `SessionOptionsImpl&`? same for RegisterCustomOpsLibrary

Reply:
should be const char*